### PR TITLE
Create community_guidelines.md

### PR DESCRIPTION
**Pull request by hollomancer**

### CODE REVIEW

File: community_guidelines.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+* Respectful and positive engagement and participation creates and fosters individual and community growth.
+* Feel free to respond to the POC from the Outreach Team who will help you connect with others and provide additional guidance or resources.
+
+**SELF-PROMOTION**
+* We value your individual journey and the projects you’re working on. Sharing your projects allows the community to assist you.
+* Submit requests for advice on #career-advice. For project promotion, you can post to #brags or #random.
+* If you are a recruiter, please be mindful about not soliciting or spamming members. You may post your positions on #job-board.
+* For code schools, please post all solicitations *ONLY* on #code-schools as this is in our Code of Conduct.
+
+**ACT PROFESSIONAL**
```
Comment (andy5995) [inline community_guidelines.md]:
```suggestion
**ACT PROFESSIONALLY**
```

File: community_guidelines.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+* Operation Code is a community of professionals. Before you post, please think about your content and be inclusive.
+* Operation Code is not Reddit, or a platform for posts that instigate others or incites violence, political ideations or affiliations.
+* As with any large group of people, disagreements will happen. We understand that you’ll have some strong opinions, but we still expect courtesy during a debate and expect you to refrain from personal attacks.
+* If we see members acting unprofessional, moderators reserve the right to remove them from the community.
```
Comment (andy5995) [inline community_guidelines.md]:
```suggestion
* If we see members acting unprofessionally, moderators reserve the right to remove them from the community.
```

I think an adverb may be more appropriate here (-ly)

File: community_guidelines.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+**RULE VIOLATIONS**
+* Being a part of the Operation Code community is a privilege - not a right. Failure to adhere to the community guidelines can result in your messages being removed (either automatically by our bot, or at an administrator’s discretion). Repeated violation will result in loss of access to the community. You are also expected to follow our Code of Conduct [ here](https://github.com/OperationCode/operationcode_docs/blob/master/community/code_of_conduct.md) .
```
Comment (andy5995) [inline community_guidelines.md:28]:
```suggestion
* Being a part of the Operation Code community is a privilege - not a right. Failure to adhere to the community guidelines can result in your messages being removed (either automatically by our bot, or at an administrator’s discretion). Repeated violation will result in loss of access to the community. You are also expected to follow our Code of Conduct [ here](https://github.com/OperationCode/operationcode_docs/blob/master/community/code_of_conduct.md).
```

**Review by andy5995 — CHANGES_REQUESTED** [review]
Looks very good. I just suggested a few minor changes.

File: community_guidelines.md (hunk 3 above)
Comment (hollomancer) [reply community_guidelines.md:28]: Thank you!

**Pull request merged**